### ISSUE TITLE
Installing and building native dependencies may fail when current directory has a space somewhere in its path

### ISSUE THREAD

**Issue by sebn** [cozy-labs/cozy-desktop#1097]
### Steps to Reproduce

1. `mkdir "foo bar"`
2. `cd "foo bar"`
3. `git clone git@github.com:cozy-labs/cozy-desktop.git`
4. `cd cozy-desktop`
5. `yarn`

### Expected behavior

Success

### Actual behavior

```
yarn install v1.5.1
[1/5] 🔍  Validating package.json...
[2/5] 🔍  Resolving packages...
[3/5] 🚚  Fetching packages...
info 7zip-bin-linux@1.2.0: The platform "darwin" is incompatible with this module.
info "7zip-bin-linux@1.2.0" is an optional dependency and failed compatibility check. Excluding it from installation.
info 7zip-bin-win@2.1.1: The platform "darwin" is incompatible with this module.
info "7zip-bin-win@2.1.1" is an optional dependency and failed compatibility check. Excluding it from installation.
[4/5] 🔗  Linking dependencies...
warning " > cozy-client-js@0.7.1" has unmet peer dependency "babel-polyfill@^6.26.0".
warning " > cozy-ui@3.0.0-beta39" has unmet peer dependency "piwik-react-router@^0.8.2".
[5/5] 📃  Building fresh packages...
[1/11] ⡀ dtrace-provider
[8/11] ⡀ elm
[3/11] ⡀ leveldown: gyp info spawn args [ 'BUILDTYPE=Release', '-C', 'build' ]
[9/11] ⡀ elm-format
error An unexpected error occurred: "/Volumes/Plus tard/cozy-desktop/node_modules/leveldown: Command failed.
Exit code: 2
Command: sh
Arguments: -c prebuild --install
Directory: /Volumes/Plus tard/cozy-desktop/node_modules/leveldown
Output:
prebuild info begin Prebuild version 4.5.0
gyp http GET https://nodejs.org/download/release/v9.11.1/node-v9.11.1-headers.tar.gz
gyp http 200 https://nodejs.org/download/release/v9.11.1/node-v9.11.1-headers.tar.gz
gyp http GET https://nodejs.org/download/release/v9.11.1/SHASUMS256.txt
gyp http 200 https://nodejs.org/download/release/v9.11.1/SHASUMS256.txt
gyp info spawn /usr/bin/python
gyp info spawn args [ '/Volumes/Plus tard/cozy-desktop/node_modules/node-gyp/gyp/gyp_main.py',
gyp info spawn args   'binding.gyp',
gyp info spawn args   '-f',
gyp info spawn args   'make',
gyp info spawn args   '-I',
gyp info spawn args   '/Volumes/Plus tard/cozy-desktop/node_modules/leveldown/build/config.gypi',
gyp info spawn args   '-I',
gyp info spawn args   '/Volumes/Plus tard/cozy-desktop/node_modules/node-gyp/addon.gypi',
gyp info spawn args   '-I',
gyp info spawn args   '/Volumes/Plus tard/cozy-desktop/node_modules/leveldown/9.11.1/include/node/common.gypi',
gyp info spawn args   '-Dlibrary=shared_library',
gyp info spawn args   '-Dvisibility=default',
gyp info spawn args   '-Dnode_root_dir=/Volumes/Plus tard/cozy-desktop/node_modules/leveldown/9.11.1',
gyp info spawn args   '-Dnode_gyp_dir=/Volumes/Plus tard/cozy-desktop/node_modules/node-gyp',
gyp info spawn args   '-Dnode_lib_file=/Volumes/Plus tard/cozy-desktop/node_modules/leveldown/9.11.1/<(target_arch)/node.lib',
gyp info spawn args   '-Dmodule_root_dir=/Volumes/Plus tard/cozy-desktop/node_modules/leveldown',
gyp info spawn args   '-Dnode_engine=v8',
gyp info spawn args   '--depth=.',
gyp info spawn args   '--no-parallel',
gyp info spawn args   '--generator-output',
gyp info spawn args   'build',
gyp info spawn args   '-Goutput_dir=.' ]
gyp info spawn make
gyp info spawn args [ 'BUILDTYPE=Release', '-C', 'build' ]
  CXX(target) Release/obj.target/leveldb/deps/leveldb/leveldb-1.19/db/builder.o
clang: error: no such file or directory: 'tard/cozy-desktop/node_modules/leveldown/9.11.1/include/node'
clang: error: no such file or directory: 'tard/cozy-desktop/node_modules/leveldown/9.11.1/src'
clang: error: no such file or directory: 'tard/cozy-desktop/node_modules/leveldown/9.11.1/deps/uv/include'
clang: error: no such file or directory: 'tard/cozy-desktop/node_modules/leveldown/9.11.1/deps/v8/include'
make: *** [Release/obj.target/leveldb/deps/leveldb/leveldb-1.19/db/builder.o] Error 1
prebuild ERR! build error 
prebuild ERR! stack Error: `make` failed with exit code: 2
prebuild ERR! stack     at ChildProcess.onExit (/Volumes/Plus tard/cozy-desktop/node_modules/node-gyp/lib/build.js:258:23)
prebuild ERR! stack     at ChildProcess.emit (events.js:180:13)
prebuild ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:209:12)
prebuild ERR! not ok 
prebuild ERR! build Error: `make` failed with exit code: 2
prebuild ERR! build     at ChildProcess.onExit (/Volumes/Plus tard/cozy-desktop/node_modules/node-gyp/lib/build.js:258:23)
```

### Informations

* **reproductible**: always
* **OS**: macOS, but possibly not specific to
* **version**: master

**Comment by sebn** [comment]
It should be fixed once we upgrade leveldown to v4.0.1: https://github.com/Level/leveldown/commit/4579bb9857583e8ed34d598a55218d10dccb4420

**Comment by taratatach** [comment]
This is now resolved.
